src/main/webapp/app/pages/curation/header/ReadOnlyBanner.tsx

```diff
@@ -0,0 +1,47 @@
+import { componentInject } from 'app/shared/util/typed-inject';
+import { Meta } from 'app/shared/model/firebase/firebase.model';
+import React, { useEffect, useState } from 'react';
+import { observer } from 'mobx-react';
+import { IRootStore } from 'app/stores';
+import { onValue, ref } from 'firebase/database';
+import { getFirebaseMetaGenePath } from 'app/shared/util/firebase/firebase-utils';
+import { BiInfoCircle } from 'react-icons/bi';
+
+export interface IReadOnlyBanner extends StoreProps {
+  hugoSymbol: string;
+}
+
+const ReadOnlyBanner = ({ isGermline, hugoSymbol, firebaseDb }: IReadOnlyBanner) => {
+  const firebaseMetaPath = getFirebaseMetaGenePath(isGermline, hugoSymbol);
```

Comment:
Please put this inside the `useEffect` and add `isGermline` and `hugoSymbol` to the dependency array.
If the path changes you'd would want the `useEffect` to be called again.